src/modules/menu.js

```diff
@@ -0,0 +1,62 @@
+import setPage from './setpage';
+
+async function fetchData() {
+  const response = await fetch('./data/menu.json', { mode: 'cors' });
```

Comment:
You might not need `cors` here since you are requesting from the same origin: https://developer.mozilla.org/en-US/docs/Web/API/Request/mode#Value, and I'm not sure but that might be the default so either way, you can omit it.